components/browser/engine-gecko-nightly/src/main/java/mozilla/components/browser/engine/gecko/GeckoEngineSession.kt

```diff
@@ -742,6 +742,10 @@ class GeckoEngineSession(
         return cookiesPolicies
     }
 
+    internal fun GeckoSession.ProgressDelegate.SecurityInformation.getIssuerName(): String? {
+            return certificate?.issuerDN?.name?.substringAfterLast("O=")?.substringBeforeLast(",C=")
```

Comment:
That is an interesting idea to parse this!

I saw that Android also has a `DName` class. I wonder if we can throw the `String` into that class and get the results back:
https://developer.android.com/reference/kotlin/android/net/http/SslCertificate.DName.html?hl=en

Reply:
Oh my gosh that is amazing, I had no idea that existed. Let me try it!